operations/ftdc.go

```diff
@@ -711,3 +713,62 @@ func fromMDB() cli.Command {
 		},
 	}
 }
+
+func toT2() cli.Command {
+	return cli.Command{
+		Name:  "t2",
+		Usage: "write data from genny output file to FTDC",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  input,
+				Usage: "source genny output data file",
+			},
+			cli.StringFlag{
+				Name:  output,
+				Usage: "write genny output data in FTDC format to this file (default: stdout)",
+			},
+		},
+		Before: requireFileExists(input, false),
+		Action: func(c *cli.Context) error {
+			inputPath := c.String(input)
+			outputPath := c.String(output)
+
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			inputFile, err := os.Open(inputPath)
+			if err != nil {
+				return errors.Wrapf(err, "problem opening file '%s'", inputPath)
+			}
+			defer func() { grip.Warning(inputFile.Close()) }()
+
+			// All genny output files are named using the workload actor and operation.
+			//   i.e., Actor.Operation.ftdc
+			//
+			// We get the actor and operation names from the ftdc filepath for use
+			// in the translation process.
+			fileName := strings.Split(inputPath, "/")
+			actorOperation := strings.Split(fileName[len(fileName)-1], ".ftdc")
+
+			var outputFile *os.File
+			if outputPath == "" {
+				outputFile = os.Stdout
+			} else {
+				if _, err = os.Stat(outputPath); !os.IsNotExist(err) {
+					return errors.Errorf("cannot write ftdc to '%s', file already exists", outputPath)
```

Comment:
is the idea here to stop if the file exists already? 
It looks to me like this will also get hit if there's some other problem with stat. Maybe we should just do this if `err == nil`?

Reply:
Pattern taken from previous implementations in that file. Also looks like its a negation of the example given here: https://pkg.go.dev/os?utm_source=gopls#IsNotExist

I think [`os.IsNotExist(err)`](https://pkg.go.dev/os?utm_source=gopls#IsNotExist) will only return true if the error argument is specifically a *PathError with Err set to [ErrNotExist](https://pkg.go.dev/io/fs#ErrNotExist). Stat will return some PathError so the following logic will check if its a "file doesn't exist" error.

Reply:
Looks like there is a handy error called [ErrExist](https://pkg.go.dev/io/fs#ErrExist) but I'm not entirely sure if there's a builtin way to expose that.